bors.toml

```diff
@@ -1,2 +1,7 @@
-status = ["Rust", "TypeScript"]
+status = [
+    'Rust (ubuntu-latest)'
```

Comment:
It is missing a comma, right ?

Reply:
Yes, good eyes!

Reply:
This seems to still be missing a comma on the latest commit (e997749a9c2537b8bd4c894aa14a8db9158a1c73)

Reply:
Must've missed a commit when bringing over changes from master, should be good now.